types/braintree-web/modules/apple-pay.d.ts

```diff
@@ -64,10 +64,32 @@ export enum ApplePayStatusCodes {
     STATUS_PIN_LOCKOUT,
 }
 
+export type ApplePayTokenizeValues = 'Yes' | 'No' | 'Unknown';
+
+export interface ApplePayDetails {
+    cardType: string;
+    cardholderName: string;
+    dpanLastTwo: string;
+    paymentInstrumentName: boolean;
+}
+
 export interface ApplePayPayload {
-    merchantIdentifier: string;
-    domainName: string;
-    displayName: string;
+    nonce: string;
+    description: string;
+    type: string;
+    consumed: boolean;
+    details: ApplePayDetails;
+    binData: {
+    commercial: ApplePayTokenizeValues;
+        countryOfIssuance: string;
+        debit: ApplePayTokenizeValues;
+        durbinRegulated: ApplePayTokenizeValues;
+        healthcare: ApplePayTokenizeValues;
+        issuingBank: ApplePayTokenizeValues;
+        payroll: ApplePayTokenizeValues;
+        prepaid: ApplePayTokenizeValues;
+        productId: string;
+    };
```

Comment:
Interesting.. did the payload change or was it just wrong before?

Reply:
It was wrong from the beginning. It wasn't used anywhere and didn't have nonce at all... So, I digged deeper and noticed that it is completely wrong...